app/Services/Media/AudioVideoSaverService.php

```
@@ -34,10 +35,25 @@ public static function saveFile(string $projectRef, mixed $file, string $fileNam
+                $maxRetries = 3;
+                $retryDelay = 1; // seconds
+
+                for ($retry = 0; $retry <= $maxRetries; $retry++) {
+                    try {
+                        $fileSaved = Storage::disk($disk)->put($targetPath, $stream, [
+                            'visibility' => 'public',
+                            'directory_visibility' => 'public'
+                        ]);
+                        break; // Success, exit retry loop
+                    } catch (Throwable $e) {
+                        if ($retry === $maxRetries || !($e instanceof S3Exception && Common::isRetryableError($e))) {
```

PHPMD (2.15.0)

Avoid using static access to class '\ec5\Libraries\Utilities\Common' in method 'saveFile'. (Clean Code Rules)

(StaticAccess)